src/Quasar/Mount/Couchbase.purs

```diff
@@ -57,26 +64,37 @@ fromJSON
   <=< decodeJson
 
 toURI ∷ Config → URI.AbsoluteURI
-toURI { host, user, password } =
+toURI { host, bucketName, password, docTypeKey, queryTimeout } =
   URI.AbsoluteURI
     (Just uriScheme)
-    (URI.HierarchicalPart (Just (URI.Authority Nothing (pure host))) Nothing)
+    (URI.HierarchicalPart
+      (Just (URI.Authority Nothing (pure host)))
+      (Just (Right (P.rootDir </> P.file bucketName))))
     (Just (URI.Query props))
   where
   props ∷ L.List (Tuple String (Maybe String))
   props = L.Nil
-    <> maybe L.Nil (\u -> pure $ Tuple "username" (Just u)) user
-    <> maybe L.Nil (\p -> pure $ Tuple "password" (Just p)) password
+    <> pure (Tuple "password" (Just password))
+    <> pure (Tuple "docTypeKey" (Just docTypeKey))
+    <> maybe L.Nil (pure <<< Tuple "queryTimeoutSeconds" <<< Just <<< show <<< unwrap) queryTimeout
```

Comment:
whenever i read unwrap being used like this it makes me wonder... `queryTimeout :: Maybe Seconds` could be replaced with anything which has a `Newtype` and `Show` instance and it would type check. maybe I'd prefer `un Seconds` here? only to increase the number of characters that would need to be changed to satisfy the typechecker whilst producing an undesired result.